pkg/controller/scyllacluster/resource_test.go

```diff
@@ -1876,6 +1876,194 @@ func TestMakeJobs(t *testing.T) {
 			},
 			expectedConditions: nil,
 		},
+		{
+			name: "cleanup job has the same placement requirements as ScyllaCluster",
+			cluster: func() *scyllav1.ScyllaCluster {
+				cluster := basicScyllaCluster.DeepCopy()
+				cluster.Spec.Datacenter.Racks[0].Placement = &scyllav1.PlacementSpec{
```

Comment:
should we have 2 racks to make sure we assign the correct, corresponding placement options?

Reply:
Good idea, added.